ui/panels-plugin/src/plugins/stat-chart/StatChartPanel.tsx

```diff
@@ -47,30 +52,68 @@ export function StatChartPanel(props: StatChartPanelProps) {
     );
   }
 
+  // Calculates chart width
+  const spacing = SPACING * (statChartData.length - 1);
+  let chartWidth = (contentDimensions.width - spacing) / statChartData.length;
+  if (isMultiSeries && chartWidth < MIN_WIDTH) {
+    chartWidth = MIN_WIDTH;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const noDataTextStyle = (chartsTheme.noDataOption.title as any).textStyle;
```

Comment:
Out of curiosity, why is the `any` needed? Could you add a clarifying comment above the eslint-disable-line?

Reply:
chartsTheme.noDataOption is type `EChartsCoreOption` but `EChartsCoreOption` doesn't have a `title` field so getting `Object is of type 'unknown'` error :(

any idea if there is a better type for noDataOption?

Reply:
We should use ECharts ComposeOption actually, that would help with tooltip type weirdness too in LineChart, there is an example internally but something we can look into in separate PR later

Reply:
ComposeOption / minimal types [docs](https://apache.github.io/echarts-handbook/en/basics/import/#creating-an-option-type-in-typescript:~:text=in%20your%20project.-,creating%20an%20option%20type%20in%20typescript,-For%20developers%20who), I had a branch where I started this I can try to find, just was going to take a bit of refactoring in LineChart but maybe easier to change in noDataOption / StatChart / GaugeChart

Reply:
oh awesome, added a new type `NoDataOption` that uses `ComposeOption` :)